flink/src/main/scala/com/metamx/tranquility/flink/BeamSink.scala

```diff
@@ -34,16 +38,24 @@ class BeamSink[T](beamFactory: BeamFactory[T])
 {
   var sender: Option[Tranquilizer[T]] = None
 
-  private val exception = new AtomicReference[Throwable]()
+  private val exception       = new AtomicReference[Throwable]()
+  private val sentCounter     = new LongCounter()
```

Comment:
In tranquility-kafka and tranquility-server, what you call "sent" and "accepted" are called "received" and "sent". Could you change these to be consistent?